tgui/packages/tgui/interfaces/OverwatchConsole.js

```diff
@@ -0,0 +1,860 @@
+              <Table.Cell>
+                <Divider />
+              </Table.Cell>
+              <Table.Cell>
+                <Divider />
+              </Table.Cell>
+              <Table.Cell>
+                <Divider />
+              </Table.Cell>
+              <Table.Cell>
+                <Divider />
+              </Table.Cell>
+              <Table.Cell>
+                <Divider />
+              </Table.Cell>
+              <Table.Cell>
+                <Divider />
+              </Table.Cell>
+            </Table.Row>
+            <Table.Row>
+              <Table.Cell>
+                <ColorBox
+                  color={squad_data.current_squad_leader_name ? 'green' : 'red'}
+                />
+                {squad_data.current_squad_leader_name ? ' Deployed' : ' None'}
+              </Table.Cell>
+              <Table.Cell>
+                <ColorBox color={squad_data.rto > 0 ? 'green' : 'red'} />
+                {squad_data.rto ? ' ' + squad_data.rto + ' Deployed' : ' None'}
+              </Table.Cell>
+              <Table.Cell>
+                <ColorBox
+                  color={squad_data.specialists > 0 ? 'green' : 'red'}
+                />
+                {squad_data.specialists ? ' Deployed' : ' None'}
+              </Table.Cell>
+              <Table.Cell>
+                <ColorBox color={squad_data.smartgun > 0 ? 'green' : 'red'} />
+                {squad_data.smartgun
+                  ? ' ' + squad_data.smartgun + ' Deployed'
+                  : ' None'}
+              </Table.Cell>
+              <Table.Cell>
+                <ColorBox color={squad_data.medics > 0 ? 'green' : 'red'} />
+                {squad_data.medics
+                  ? ' ' + squad_data.medics + ' Deployed'
+                  : ' None'}
+              </Table.Cell>
+              <Table.Cell>
+                <ColorBox color={squad_data.engineers > 0 ? 'green' : 'red'} />
+                {squad_data.engineers
+                  ? ' ' + squad_data.engineers + ' Deployed'
+                  : ' None'}
+              </Table.Cell>
+              <Table.Cell>{squad_data.alive}</Table.Cell>
+              <Table.Cell>{squad_data.total_deployed}</Table.Cell>
+            </Table.Row>
+          </Table>
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+export const OverwatchSupplies = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { x_supply, y_supply, supply_cooldown, supply_ready } = data;
+
+  const cooldown = round(supply_cooldown / 10);
+
+  const [target_x, setTargetX] = useLocalState(context, 'target_x', x_supply);
+  const [target_y, setTargetY] = useLocalState(context, 'target_y', y_supply);
+
+  return (
+    <>
+      <ProgressBar
+        ranges={{
+          good: [-Infinity, 200],
+          average: [200, 400],
+          bad: [Infinity, 400],
+        }}
+        minValue={0}
+        maxValue={500}
+        value={cooldown}>
+        {cooldown + ' seconds'}
```

Comment:
you don't need to cat the strings together, seconds can be outside of the braces